integration_test/tests/wallet.rs

```diff
@@ -625,6 +625,24 @@ fn wallet__list_descriptors() {
     assert!(has_descriptor, "No standard descriptors found in listdescriptors result");
 }
 
+#[test]
+fn wallet__list_lock_unspent__modelled() {
+    let node = Node::with_wallet(Wallet::Default, &[]);
+    node.fund_wallet();
+
+    let utxos: ListUnspent = node.client.list_unspent().expect("listunspent");
+    let utxo = utxos.0.first().expect("need at least one utxo to lock");
+    let txid = utxo.txid.parse::<bitcoin::Txid>().expect("parse txid");
+    let vout = utxo.vout as u32;
+    node.client.lock_unspent(&[(txid, vout)]).expect("lockunspent");
+
+    let json: ListLockUnspent = node.client.list_lock_unspent().expect("listlockunspent");
+    let model: Result<mtype::ListLockUnspent, ListLockUnspentItemError> = json.into_model();
+    let model = model.unwrap();
+
+    assert!(model.0.iter().any(|o| o.txid.to_string() == utxo.txid && o.vout as i64 == utxo.vout));
```

Comment:
Shouldn't need to allocate a string here. In general we only want to allocate if totally necessary. (This is a bit of geek thing, we always tend to optimize for speed even in test code.)

Reply:
Fixed by addressing the previous comment.